### ISSUE TITLE
_default method

### ISSUE THREAD

**Issue by HoneyryderChuck** [samlown/translate_columns#2]
Greetings,

I installed the plugin, did everything i was supposed to do, except for providing me with a "_default" method to the translatable fields. Well, I see that in the description you've made of the plugin, it was at least supposed to exist (title_default in that case). As for what I do in my translatable model, here's the print:

has_many :translations, :class_name => 'ObjectTranslation'
translate_columns :description

so basically, i was supposed to have a "description_default" method attached. 

P.S.: Are you planning to find a way of defining the ObjectTranslation class automagically? I had to create the model and the migrations for it myself, which i don't find that pretty. 

Best Regards,
Tiago
